File: 1800-1899/1829.c
#include <stdio.h>
int main() {
	int n;
	while(scanf("%d",&n)!=EOF) {
		int a=1,b=1,k;
		while(b<n) {
			k=b;
			b=a+b;
			a=k;
		}
		if(b==n) printf("mwy win\n");
		else printf("sjy win\n");
	}
}

File: 1800-1899/1829.py
F=[0,0,0]
def init():
    F[1]=1
    F[2]=1
    for i in range(3,30):
        F.append(F[i-1]+F[i-2])
init()
while True:
    try:
      n=int(input())
      ans="sjy win"
      if n in F:
          ans="mwy win"
      print(ans)
    except:
        break
